openedx/features/caliper_tracking/tests/current/edx.course.enrollment.deactivated.json

```diff
@@ -0,0 +1,25 @@
+{
+   "username":"honor",
```

Comment:
`username` value can be assigned to `name` key in `actor` object.

Reply:
Done.